pkg/cmd/gist/view/view_test.go

```diff
@@ -200,12 +217,17 @@ func Test_viewRun(t *testing.T) {
 		io.SetStdoutTTY(true)
 		tt.opts.IO = io
 
-		tt.opts.Selector = "1234"
+		if tt.opts.Selector == "" {
+			tt.opts.Selector = "1234"
```

Comment:
Instead of adding this fallback here, which isn't obvious, could you add `Selector = "1234"` to all existing test cases instead?

Reply:
Sure.

I've found that gist URLs without username like https://gist.github.com/baf3ae0ee13b1f6846e6dea88bdfdc39 are valid too so I'll have to refactor this code and add some tests.